pscript/stdlib.py

```diff
@@ -141,6 +141,14 @@ def get_all_std_names():
     }
 }"""
 
+FUNCTIONS['create_set'] = """function () {
+    var s = new Set();
+    for (var i=0; i<arguments.length; i+=2) {
+        if (!s.has(arguments[i]) && !FUNCTION_PREFIXop_contains(arguments[i], s)) { s.add(arguments[i]) };
```

Comment:
Flake: too long lines
```suggestion
        if (!s.has(arguments[i]) && !FUNCTION_PREFIXop_contains(arguments[i], s)) {
            s.add(arguments[i])
        };
            
```